src/Pages/ClassDetail/ClassDetail.js

```diff
@@ -1,9 +1,193 @@
+          <LeftSide>
+            <TitleContainer>
+              <div className='titleBox'>
+                <span className='title'>{detaildata.class_title}</span>
+              </div>
+              <div className='lecture'>
+                <div className='lectureTitle'>
+                  {detaildata.lecture_title}
+                  <span>
+                    <IoIosArrowDown size='25' />
+                  </span>
+                </div>
+                <div className='page'>
+                  <div>
+                    <div className='previousPage'>
+                      <BsArrowLeft />
+                      이전 콘텐츠
+                    </div>
+                  </div>
+                  <div>
+                    <div className='nextPage'>
+                      다음 콘텐츠
+                      <BsArrowRight />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </TitleContainer>
+            <VideoContainer>
+              <iframe
+                title='hs'
+                width='830'
+                height='504'
+                src='./images/HS/해수.mp4'
+                frameborder='0'
+                allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
+                allowfullscreen
+                alt='video'
+              />
+            </VideoContainer>
+            <CommentBox>
+              <CommentHeader>
+                <span className='comment'>댓글</span>
+                <span className='length'>15개</span>
+              </CommentHeader>
+              <CommentInput />
+            </CommentBox>
+          </LeftSide>
+        </LeftContainer>
+        <ClassRightSide
+          detaildata={detaildata}
+          lecture_detail={detaildata.lecture_detail}
+        />
+      </Listpage>
+    </div>
+  );
+};
 
 export default ClassDetail;
+
+const Listpage = styled.div`
+  display: flex;
+  width: 100%;
+`;
+
+const LeftContainer = styled.div`
+  width: 67%;
+  height: 100vh;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  padding: 32px;
+`;
+
+const LeftSide = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const TitleContainer = styled.div`
+  width: 850px;
+  box-sizing: border-box;
+  padding: 0px 12px;
+  .titleBox {
+    width: 100%;
+    font-size: 14px;
+    letter-spacing: -0.15px;
+    color: #1b1c1d;
+    margin-bottom: 12px;
+    .title {
+    }
+  }
+  .lecture {
+    display: flex;
+    justify-content: space-between;
+
+    .lectureTitle {
+      position: relative;
+      font-size: 24px;
+      line-height: 34px;
+      letter-spacing: -0.4px;
+      font-weight: 700;
+      span {
+        margin-left: 10px;
+        position: absolute;
+        top: 3px;
+      }
+    }
+    .page {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      div {
+        display: flex;
+        justify-content: center;
+        align-items: center;
```

Comment:
혜수님 아래와 같은 방식으로 extend 할 수 있답니다. 한번 사용해보세요 ~

```suggestion
const displayCenter = styled.css`
        display: flex;
        justify-content: center;
        align-items: center;
`

const Container = styled.div`
 ${displayCenter}
`
  
```